cmd/storage/postgresql/postgresql_init.go

```diff
@@ -0,0 +1,72 @@
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+type PostgreSQL struct {
+	Address  string
+	UserName string
+	Password string
+	DBName   string
+	dbConn   *sql.DB
+}
+
+func (db *PostgreSQL) Init() error {
+	var err error
+	ps := fmt.Sprintf("host=%s user=%s password=%s database=%s sslmode=disable",
+		db.Address, db.UserName, db.Password, db.DBName)
+
+	db.dbConn, err = sql.Open("pgx", ps)
+	if err != nil {
+		return err
+	}
+
+	db.dbConn.Exec("DROP TABLE orders;")
+	db.dbConn.Exec("DROP TABLE users;")
+	db.dbConn.Exec("DROP TABLE order_spend;")
+	db.dbConn.Exec("DROP TYPE status_enum;")
+
+	_, err = db.dbConn.Exec(`CREATE EXTENSION pgcrypto;`)
```

Comment:
```suggestion
	_, err = db.dbConn.Exec(`CREATE EXTENSION IF NOT EXISTS pgcrypto;`)
```
https://www.postgresql.org/docs/current/sql-createextension.html